From 922a38c8a9900bc7920b836256a6102d5dc14c86 Mon Sep 17 00:00:00 2001
From: codeworkx <daniel.hillenbrand@codeworkx.de>
Date: Wed, 22 Aug 2018 14:24:30 +0200
Subject: [PATCH 205/206] Snap: use platform cert

This is needed to be able to put Snap into it's own selinux domain.

Change-Id: I23f73723a9b62ee67b6d0f234898828eba945962
---
 Android.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 1f0a6e32e..e74c8c0cb 100644
--- a/Android.mk
+++ b/Android.mk
@@ -35,7 +35,8 @@ LOCAL_JAVA_LIBRARIES := org.lineageos.platform.internal
 
 LOCAL_PACKAGE_NAME := Snap
 LOCAL_PRIVILEGED_MODULE := true
-LOCAL_PRIVATE_PLATFORM_APIS:=true
+LOCAL_PRIVATE_PLATFORM_APIS := true
+LOCAL_CERTIFICATE := platform
 
 LOCAL_AAPT_FLAGS += --rename-manifest-package org.lineageos.snap
 
-- 
2.17.1

